src/main/webapp/index.html

```diff
@@ -2,18 +2,92 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Political Contributions Tracker Table Page</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Political Contributions Tracker</h1>
+
+      <p>Welcome to the prototype of the Political Contributions Tracker.
+        <br>
```

Comment:
Can you add a TODO to update this part for the coming week?

Reply:
@Priya411 Done, please take a look at lines 13, 25, and 36 in index.html for the added TODO comments.